src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -484,11 +485,17 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this.turndownService.addRule('a', {
 			filter: 'a',
 			replacement: (content, node) => {
-				if (node?.href) {
-					let relativePath = this.findPathRelativeToContent(node.href);
-					if (relativePath) {
-						return `[${node.innerText}](${relativePath})`;
-					}
+				//if notebook is untrusted then the link is save on the title
+				let path: URI;
+				if (node.href) {
+					path = URI.parse(node.href);
+				} else {
+					path = URI.file(node.title);
```

Comment:
Why are you using URI.file here?

Reply:
If I use URI.parse, then it doesn't create the the path correctly.

Reply:
Why? Shouldn't they be the same value - just stored in different attributes?